custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/impl/JenkinsWarPatcher.java

```diff
@@ -189,21 +188,21 @@ private void excludeLib(DependencyInfo lib) throws IOException, InterruptedExcep
         }
     }
 
-    public JenkinsWarPatcher addHooks(@Nonnull Map<String, File> hooks) throws IOException {
-        for (Map.Entry<String, File> hookSrc : hooks.entrySet()) {
-            final String hookId = hookSrc.getKey();
-            GroovyHookInfo hook = config.getHookById(hookId);
-            if (hook == null) {
-                throw new IOException("Cannot find metadata for the hook with ID=" + hookId);
+    public JenkinsWarPatcher addResources(@Nonnull Map<String, File> resources) throws IOException {
```

Comment:
Just generalized logic so we can easily append more resource types to the WAR file if needed